ntoskrnl/ex/uuid.c

```diff
@@ -34,6 +34,7 @@
 #pragma alloc_text(INIT, ExLuidInitialization)
 #endif
 
+C_ASSERT(sizeof(UUID_CACHED_VALUES_STRUCT) - FIELD_OFFSET(UUID_CACHED_VALUES_STRUCT, ClockSeqHiAndReserved) >= sizeof(((GUID *)0)->Data4));
```

Comment:
`RTL_FIELD_SIZE(GUID, Data4)` would make the second part more readable.
I also personally liked this better inside the function close to where we make use of this (in)equality fact.